cluster/olm/clusterserviceversion.yaml.tmpl

```diff
@@ -14,11 +14,26 @@ metadata:
       data:
         token: "Output of `up cloud controlplane attach | base64` command"
     operators.operatorframework.io/internal-objects: '["locks.pkg.crossplane.io"]'
-    categories: "Application Runtime, Cloud Provider, Database, Integration & Delivery, Networking, Storage"
     capabilities: Seamless Upgrades
-    repository: https://github.com/upbound/universal-crossplane
+    categories: "Application Runtime, Cloud Provider, Database, Integration & Delivery, Networking, Storage"
+    containerImage: upbound/uxp-bootstrapper
+    createdAt: "2021-05-18T00:00:00+0000"
```

Comment:
Will this get updated dynamically?

Reply:
No, it's the creation time of the operator entry rather than the version so it'll always stay like that.

Reply:
> it's the creation time of the operator entry

Right, shouldn't this be set to the time when we run `make olm` (or once #125 is implemented, at build time)? This is kind of an unfortunate field to be part of the CSV schema because timestamps break reproducible builds :/ Anyway, happy to address in #125 and we can just update manually for now 👍

Reply:
Not really, it's the date of the [very first release](https://github.com/operator-framework/community-operators/blob/master/docs/packaging-required-fields.md#required-fields-for-operatorhub) on OperatorHub. Take a look at other entries like [ArgoCD](https://operatorhub.io/operator/argocd-operator) or [Couchbase Operator](https://operatorhub.io/operator/couchbase-enterprise).

Reply:
Oh -- that is very interesting 🤔